noxfile.py

```diff
@@ -16,11 +16,9 @@ def build(session, autobuild=False):
     shutil.rmtree("build", ignore_errors=True)
 
     if autobuild:
-        command = "sphinx-autobuild"
+        session.run("sphinx-autobuild","-H0.0.0.0", "-W", "-b", "html", "source", "build")
     else:
-        command = "sphinx-build"
-
-    session.run(command, "-W", "-b", "html", "source", "build")
+        session.run("sphinx-build", "-W", "-b", "html", "source", "build")
```

Comment:
Here's how you can do this without having to copy-paste that `session.run` multiple times:
```suggestion
        command = "sphinx-autobuild"
        extra_args = "-H", "0.0.0.0"
    else:
        command = "sphinx-build"
        extra_args = ()

    session.run(command, *extra_args, "-W", "-b", "html", "source", "build")
```

Pro tip: you can accept this suggested change by clicking the button below 👇